src/sidebar/components/Chat/MessageGroup/index.tsx

```
@@ -0,0 +1,179 @@
+    onContinueGenerating,
+    onRegenerate
+  }: MessageGroupProps) => {
+    const { t } = useTranslation()
+    const parsedContent = parseMessageWithContext(node.content)
+
+    const message =
+      node.role === 'assistant' ? (
+        preprocessMarkdown(node.content)
+      ) : (
+        <React.Fragment key={`message-${node.id}`}>
+          <Markdown children={parsedContent.message} />
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)